analyze.js

```diff
@@ -55,15 +55,18 @@ async function prepareInputsZip(inputsGlob, targetFile) {
   const output = fs.createWriteStream(targetFile);
   const archive = archiver('zip');
   archive.on('end', () => core.info("Finished writing ZIP"))
-  archive.on('warning', (err) => core.warning("Warning when writing ZIP: ", err))
-  archive.on('error', (err) => core.error("Error when writing ZIP: ", err))
+  archive.on('warning', (err) => core.warning("Warning when writing ZIP: " + err))
+  archive.on('error', (err) => core.error("Error when writing ZIP: " + err))
 
   archive.pipe(output);
 
   let numWritten = 0
   const workingDir = process.cwd()
   for await (const file of inputFilesGlob.globGenerator()) {
-    archive.file(makeRelative(workingDir, file))
+    const relPath = makeRelative(workingDir, file)
+    if (file == targetFile || relPath == targetFile) continue
```

Comment:
Prevent us from attempting to include the ZIP within the ZIP

There doesn't appear to be a known cross-platform way of working with temp directories for GitHub Actions, so we just keep the ZIP in the workspace root since we at least know we have permissions to access that